resources/assets/js/components/FindSkillsModal.tsx

```diff
@@ -0,0 +1,481 @@
+                                  messages.disabledSkillButton,
+                                )}
+                              </span>
+                            </button>
+                          ) : (
+                            <button
+                              data-h2-button=""
+                              data-h2-grid-item="b(1of4)"
+                              type="button"
+                              disabled={isOldSkill}
+                              onClick={() => {
+                                // If the skill has been selected then remove it.
+                                // Else, if the has not been selected then add it to addedSkills list.
+                                if (isAdded) {
+                                  setNewSkills(
+                                    newSkills.filter(
+                                      (applicantSkill) =>
+                                        applicantSkill.id !== skill.id,
+                                    ),
+                                  );
+                                } else {
+                                  setNewSkills([...newSkills, skill]);
+                                }
+                              }}
+                            >
+                              <p
+                                data-h2-button-label
+                                data-h2-font-weight="b(700)"
+                                data-h2-font-style="b(underline)"
+                                data-h2-font-color={`${
+                                  isAdded ? "b(theme-1)" : "b(black)"
+                                }`}
+                              >
+                                {isAdded
+                                  ? intl.formatMessage(
+                                      messages.removeSkillButton,
+                                    )
+                                  : intl.formatMessage(
+                                      messages.selectSkillButton,
+                                    )}
+                              </p>
+                            </button>
+                          )}
+                        </li>
+                      );
+                    })}
+                  </ul>
+                ) : (
+                  <p>{intl.formatMessage(messages.noSkills)}</p>
+                )}
+              </div>
+            )}
+          </div>
+        </Dialog.Content>
+        <Dialog.Actions
+          data-h2-grid="b(middle, expanded, padded, .5)"
+          data-h2-margin="b(all, 0)"
+          data-h2-bg-color="b(gray-1, 1)"
+        >
+          <div data-h2-align="b(left)" data-h2-grid-item="b(1of2)">
+            <Dialog.ActionBtn
+              buttonStyling="stop, round, solid"
+              data-h2-padding="b(rl, 2) b(tb, .5)"
+              data-h2-bg-color="b(white, 1)"
+              onClick={() => {
+                setSkillsResults([]);
```

Comment:
I'm having some trouble testing the cancel button, but I think this should do all the same things as the Back button?

Reply:
This was addressed in the hydrogen slack channel. It's the same issue we had with the accordion event listener. It should be resolved in the next release.